### ISSUE TITLE
Bulk create doesn't return json of created records

### ISSUE THREAD

**Issue by danielwanja** [drogus/bulk_api#15]
The update, put "/api/bulk, returns all updated records. The create, post "/api/bulk", returns only the first and not all created records. This is an issue as we need the ids and other attributes that Rails assigned during the create.

In engine_spec.rb I've added these two tests. The create fails, the update works.

```
it "should create given records and return all created records" do
  post "/api/bulk", { :tasks => [{:title => "New1"}, {:title => "New2"}] }
  body = JSON.parse(last_response.body)
  body["tasks"].length.should == 2
end

it "should update given records and return all updated records" do
  task = Task.create(:title => "Bar")
  put "/api/bulk", { :tasks => [{:title => "Foo2", :id => @task.id}, {:title => "Bar2", :id => task.id}]}

  @task.reload.title.should == "Foo2"
  task.reload.title.should == "Bar2"

  body = JSON.parse(last_response.body)
  body["tasks"].length.should == 2
  body["tasks"][0]["title"].should == "Foo2"
  body["tasks"][1]["title"].should == "Bar2"

end
```

**Comment by drogus** [comment]
As stated in readme, request to create need to have _local_id attribute, otherwise you will not have track of which record you get in response for sproutcore, which will lead to inconsistencies in data store.

**Comment by danielwanja** [comment]
I understand the use of the _local_id_, especially in case of validation errors. However if you create five new records how to get their id once they have been created if the post doesn't return the new attributes and id?  How do you deal with the scenario I create 5 new records, create them, then do changes, and update them? When is the datastore update with the just created ids?

**Comment by drogus** [comment]
local_ids are basically guids from sproutcore, so the flow in such situation should go like this:
- send request to create with local_ids
- get response and update all of the saved records with new id from rails (like here: https://github.com/drogus/bulk_data_source/blob/master/frameworks/foundation/data_sources/bulk.js#L296 )
- at this point records in store should already have an id
- after doing changes to those records now, data source should send request to update, local_id can be ommited now

Is that information that you need or have I missed something?

**Comment by danielwanja** [comment]
That's exactly what I was expecting. But in my tests I don't get all the record information back from the create but only the first record. So If I save two records I just get the json for one back. See the it "should create given records and return all created records" test it fails and returns only one instead of two records.

I'm creating a Flex library that does similar work (https://github.com/danielwanja/bulk_data_source_flex) than your javascript library and that's when I noticed this issue. I went to the Rails code and added the test to confirm the issue.
